interop/client/client.go

```diff
@@ -37,7 +37,8 @@ import (
 )
 
 const (
-	googleDefaultCredsName = "google_default_credentials"
+	googleDefaultCredsName        = "google_default_credentials"
+	computeEngineChannelCredsName = "compute_engine_channel_creds"
```

Comment:
I assume "channel" is used here for interop compatibility across languages.  If that's the case, maybe still `s/Channel//` in the code.

It's a little unfortunate that this option doesn't match `"google_default_credentials"` (no "channel", and "creds" has been abbreviated), but :man_shrugging:.

Reply:
Makes sense, I made the `s/Channel//` change throughout the code, so that the only thing with "channel" in the name is the name of the test case and the `custom_credentials_type` option.

Personally I prefer to refer to this credentials type, across languages in general, as "compute engine channel credentials", at least in documentation and the test case name. Because "compute engine credentials" (without "channel") IMO is already taken as a name.

I agree that `creds` should probably be expanded out to `credentials`, though. That change will require changes outside of this PR though.

Reply:
> I agree that creds should probably be expanded out to credentials, though. That change will require changes outside of this PR though.

I don't *really* care, since it's just some flags in a binary we use internally. 
It's just unfortunate that we ended up with "Google Default Creds" and "Compute Engine **Channel** Creds", when they're both essentially the same thing.